var/spack/repos/builtin/packages/root/package.py

```diff
@@ -32,6 +32,8 @@ class Root(CMakePackage):
 
     homepage = "https://root.cern.ch"
     url      = "https://root.cern.ch/download/root_v6.09.02.source.tar.gz"
+    list_url = "https://root.cern.ch/download/"
```

Comment:
@paul-chelarescu I think this should not be on this PR.

Reply:
Correct